DistributedCollective/Sovryn-frontend
# change minReturn with minEntryPrice

To Test: 
margin trading - try to open new trades, increase position to already opened positions
lending - try on depositing to lending pool
All transactions should pass

updated QA for margin trading:
try all options when opening margin trading for pair:
- btc long/short
- xusd long/short
if dialog says that tx will fail - try increasing  slippage and decreasing deposit amount;
if dialog says that tx will fail with 1% slippage and minimal deposit amount - leave screenshot as a comment showing error message and options used to open position.
